actuary.go

```diff
@@ -42,19 +43,40 @@ func (l *ContainerList) Running() bool {
 	return false
 }
 
-func CreateContainerList(c *client.Client) (l ContainerList) {
+type Target struct {
+	Client     *client.Client
+	Info       types.Info
+	Containers ContainerList
+}
+
+func NewTarget() (a Target, err error) {
+	a.Client, err = client.NewEnvClient()
+	if err != nil {
+		fmt.Printf("Unable to create Docker client")
```

Comment:
I 'll probably avoid using `fmt.Printf()` and switch to `log` instead which is already widely used in actuary.